### DISCUSSION TITLE
Adjust system arrival point's and intersystem travel mechanics

### DISCUSSION THREAD

**Discussion by xX-Dillinger-Xx** [endless-sky/endless-sky#9861] [Balance, Mechanics, and Interactions]
I'm expanding on an idea that has been mentioned before. I believe that arrival points of some systems need adjusting to allow for a more immersive experience. In fact the whole intersystem travel mechanics could use some tweaks. Most of my concerns revolve around logic, believability and frustration during game play.
This idea has the potential to solve the issue presented in #9802 and #9851 (i.e. battles in human space are to short) 

**The issues with the current setup:**
- ships arrive in the middle of an on going battle -- logic issue and frustration
- using a jump drives, escorts arrive scattered all over the place. -- logic issue
- ships launch from the surface straight into a battle -- logic issue and frustration

Here is the problem with the in game logic(characters perspective). 
Every race has achieved universe wide communication as well as highly advanced sensors and short range scanners, but nobody has invented long range scanners or detection devices. (no i don't want a new outfit) The navigation systems are advanced enough to avoid hitting asteroids and other ships while traveling though hyper space but, some how can't detect a battle and determine a safe place to arrive when entering a system. 
Apparently, spaceports don't have any launch control or we would be not be able to launch into the middle of a death ball. 
 
Yes, I know there are times where battles upon launch are part of a storyline and I'm not referring to them or the lore that pirates or others are hanging around waiting to attack. Overrides, could be implemented to allow for the lore of attacks on planets etc.

There are 3 main variables to consider
- system arrival points for the player, any escorts, and any ships being escorted. Along with tighter fleet formation while arriving
- system arrival points for the NPC
- system spawn points for NPC

**Thoughts on solutions:**
- make arrival points for the player, in some systems, further out from center and away from battles. 
  - behind the scenes, and therefore unseen by the player, the system is divided into player and escorts arrival zones and NPC arrival zones. Close to each other but not on top of each other. Spawn points for NPC's would also be in the NPC arrival zones.
  -  PRO: allows player assessment of the situation and reaction time, makes ship speed and maneuverability more relevant, is more immersive, feels more ~real~ believable.
- tighten up the fleet formation while jumping 
  - logically, navigational control should be able to maintain a tighter formation while arriving. Hyper drive is better but could be tighter, and  jump drive makes no sense. 
  - PROS: adds believability, supports fleet play style without effecting soloists, helps escort missions, and allows for better tactics. 
    - If I'm supposed to be escorting something, shouldn't I stick close to the escorted? (The whole escorting missions system needs tweaking, but that's a separate issue.) 
- launching -- Have spawn points for NPC a little further away from the planets so the player has less of a chance of launching into a death ball. Battles will still happen but not right on top of the planet. 
  - PROS: provides a little assessment and reaction time.

I, as well as, a few others have played around with the arrival points within the confines of the current mechanics. In other words editing TXT files. It does make a positive improvement to game play IMO.

**Comment by Hurleveur** [comment]
I actually like that escorts arrive scattered with a jump drive, it gives you incentive to use hyperdrives and makes fighting more interesting.
I do agree that it's a bit much right now, maybe we should have a way to control it and make the current scatter only happen in the Ember Waste.

I honestly dont know why your escorts would be able to target where exactly they're going in a system with a JD they barely understand, gameplay should be the focus anyway

**Reply by xX-Dillinger-Xx** [reply]
I know 'fleet' play isn't a big focus. I hate the fact that my escorts land all over the place. Because I use small escorts and not a lot of them, it hurts too much to see them start dropping like flies. There is always greater strength in numbers. Maybe, after an AI update were the AI looses it's suicidal tendencies. Maybe, pilots should stop signing up for Kamikaze training. 

>I honestly dont know why your escorts would be able to target where exactly they're going in a system with a JD they barely understand

It wouldn't be the first or last thing we need to just take on faith. 😏  But, like you said, gameplay first.

**Reply by Hurleveur** [reply]
Just use hyperdrives? (And escorts having most dps from weaps w more than  1k range, or 500 homing will keep their distances)

**Reply by xX-Dillinger-Xx** [reply]
Some of my ships already have both HD and JD.
Yeah. I agree and already value distance weapons more then DPS but hate missiles, rockets etc.

**Reply by Hecter94** [reply]
> I do agree that it's a bit much right now, maybe we should have a way to control it and make the current scatter only happen in the Ember Waste.

I'd like to see the scatter depend on the size of your fleet. With just a few ships, your fleet should jump in a relatively dense formation, however as you start adding more ships, they begin to spread out further, with extremely large fleets subject to a large amount of spread across the entire system.

**Comment by MasterOfGrey** [comment]
> Every race has achieved universe wide communication as well as highly advanced sensors and short range scanners, but nobody has invented long range scanners or detection devices.

It's probably worth noting that this isn't strictly true. In fact, connecting Hai space and Human space communications is a significant plot point in the Hai campaign, and the lack of communications in some parts is noted in FW and a plot element of the planned story for the Republic.
Humans (and Hai) are dependent on space relays to exploit hyperlanes for sending communications at pseudo-instantaneous speeds.

Also, given the foreshortened scale of the systems, the short range scanners are, in effect, long range scanners.

This pulls apart one particular point:

> The navigation systems are advanced enough to avoid hitting asteroids and other ships while traveling though hyper space but, some how can't detect a battle and determine a safe place to arrive when entering a system.
> [...]
> logically, navigational control should be able to maintain a tighter formation while arriving. Hyper drive is better but could be tighter, and jump drive makes no sense.

Hyperdrive initiation means that you're a) disconnected from the comms network during jump, and b) temporarily blind as you effectively travel ftl
There really is no way for you to detect a battle that (accounting for experiential vs actual time) starts after you leave the previous system.
But at least with Hyperdrive you arrive _approximately_ in formation.

As for Jump drive, it was fine that you were scattered when all ships arrived right next to the star. Now that arrival distances exist/ if/when/as arrival distances are implemented, it presents a major tactical hazard.
It would make sense to change how arriving via jump drive occurs so you at least have all ships arrive within some arc facing your approach vector.
You're transiting through another layer of reality, so there's no hope of seeing ahead of yourself, but you're still doing so with gravity as a basis for navigation - so your ships should really all arrive in _at least_ the same quadrant of the system.

**Reply by xX-Dillinger-Xx** [reply]
Everything you quoted was a just a guess as to what I thought must be happening from an in game perspective. It seems that there is already a whole explanation in place. So just ignore the made up parts. That being said, I think you get my point of the rest of it.
